Gruntfile.js

```diff
@@ -527,8 +516,6 @@ module.exports = function(grunt) {
     'eslint'
   ]);
   grunt.registerTask('prepareDeploy', [
-    'package',
-    'compress:deployFirebaseCode',
```

Comment:
Previously Travis used some clever auto-unzipping protocol to upload the files. I think it was possibly a HTTP post. Now we use rsync which means that we need the files that will be uploaded in their original form, not zipped.